client/client.go

```diff
@@ -137,7 +137,19 @@ func (client *Client) List(absolutePath string) (result []string, err error) {
 	if client.isTopLevelPath(absolutePath) {
 		result = client.listTopLevel()
 	} else {
-		result, err = client.listLowLevel(normalizedVaultPath(absolutePath))
+		result, err = client.listLowLevel(normalizedVaultPath(absolutePath), MODE_DIRECTORY_ONLY)
+	}
+	return result, err
+}
+
+// List elements at the given absolutePath, using the given client
+func (client *Client) ListAll(absolutePath string) (result []string, err error) {
```

Comment:
Lets move that to `Run` function of `lsall.go`.

Reply:
That doesn't work without changing the package structure. 

The used functions as `isTopLevelPath`, `topLevelTraverse` and `lowLevelTraverse` are not available in the
lsall.go `cli` package. So without a package refactoring not really possible.
And the other command functions like `List`, `Delete`, ... are also defined here and not in there command `Run` function.